snippets/binarySearch.md

```diff
@@ -0,0 +1,23 @@
+### binarySearch
+
+Use recursion to perform a logarithmic search similar to `.indexOf()` that finds the index of a value within an array. The two differences being 
+1. Operation only works with sorted arrays
```

Comment:
Only arrays sorted in ascending order, if I understand this correctly.

Reply:
Yea either or, can be ascending or sorted